.github/workflows/continous-integration.yaml

```diff
@@ -0,0 +1,111 @@
+name: Continous Integration
+on: [push, pull_request]
+
+env:
+  CI: true
+  NODE_OPTIONS: --max-old-space-size=4096
+
+jobs:
+  test:
+    name: Test
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [12, 14]
```

Comment:
Can you please add Node.js 10 to our test matrix?

```suggestion
        node-version: [10, 12, 14]
```

The release line 10.x is supported until April 2021 AFAICT:

<img width="850" alt="Screen Shot 2021-01-06 at 16 07 39" src="https://user-images.githubusercontent.com/1140553/103783681-52f3cf00-5039-11eb-96fc-b3ad092265ee.png">


See https://github.com/nodejs/Release

Reply:
Added